packages/nimble-components/src/wafer-map/tests/worker-renderer.spec.ts

```diff
@@ -0,0 +1,42 @@
+import type { Remote } from 'comlink';
```

Comment:
Lets update the file `create-matrix-renderer.spec.ts`

Reply:
updated!
![image](https://github.com/ni/nimble/assets/110180309/885f47e2-c6f9-4865-b190-9ab7006fc9b1)